pwndbg/commands/ida.py

```diff
@@ -110,20 +112,23 @@ def save_ida():
     # Remove old version
     os.unlink(full_path)
 
+
 class ida(gdb.Function):
     """Evaluate ida.LocByName() on the supplied value.
     """
+
     def __init__(self):
         super(ida, self).__init__('ida')
+
     def invoke(self, name):
-        name   = name.string()
+        name = name.string()
         result = pwndbg.ida.LocByName(name)
 
-        if 0xffffe000 <= result <= 0xffffffff \
-        or 0xffffffffffffe000 <= result <= 0xffffffffffffffff:
+        if 0xffffe000 <= result <= 0xffffffff or 0xffffffffffffe000 <= result <= 0xffffffffffffffff:
```

Comment:
It looks like we hit E122 the way it is.  From grepping for other E122 errors, it looks like it's not something we really want anyway, since it complains profusely about `dict` and `list` ordering, e.g.: https://github.com/pwndbg/pwndbg/blob/master/pwndbg/__init__.py#L76